### DISCUSSION TITLE
Questions Regarding Project Goals and Potential Implementations

### DISCUSSION THREAD

**Discussion by guilherme-mertens** [akdasUAF/ForestFireDetection#30] [General]
Hi! As a former perception analyst for an autonomous racing car, I find this project intriguing and in line with my career goals, prompting me to make my application.

1. Of the main objectives is to deploy this model onto a drone platform. Could you provide insights into the camera specifications employed by these drones? Utilizing stereo cameras could be advantageous for certain algorithms :)
2. I have access to a Jetson Tx2 board and intend to conduct tests on neural networks that I'll be training in the coming days. Are there established requirements for the minimal operating frequency of the neural networks?
3. What additional data points would be pertinent to extract from the images? I thought of real dimensions of the fire, predictive analysis regarding the fire's propagation direction, assessment of fire risk based on environmental parameters, among others.
4. Have you conducted any evaluations on semantic segmentation models thus far?

Appreciate your prompt response.

**Comment by YaliWang2019** [comment]
Hi there! Sorry for the late reply. First, in short-term view, if the drone’s flight controller is not connected with Jetson Nano, then we’re not using the drone’s camera but using a USB camera  connected to the Jetson Nano. There’re many high-resolution USB cameras available on the market that are compatible with Jetson Nano. The basic specifications of the camera should be at least working on the Jetson Nano with relatively high resolution. Second, Jetson TX2 is a good choice, as it’s also a low-energy edge device (I checked it’s 7.5 W right?) Third, we shall get some results like estimations of the fire spread characteristics (e.g., speed, directions, etc.) Lastly, we haven’t reached out to semantic segmentation models yet, but we did many research on object detection models like YOLO and D2GO.
